src/phase0/presets/mainnet.rs

```diff
@@ -95,7 +97,7 @@ pub const PRESET: Preset = Preset {
 };
 
 pub fn context() -> Context {
-    Context::with_preset(&PRESET)
+    Context::from(&PRESET, &CONFIG)
 }
```

Comment:
following the symmetry here, there should be a `phase0::state_transition::context::mainnet` module and this should not live in the `preset`s

i'll make a note to fix this outside of this PR, it isn't super urgent